### PR TITLE
Fix JavaDoc Generation

### PR DESCRIPTION
**Pull request by wilsonwatson**
1. Fixed javadoc generation failure with `SwerveModule.java` (change `<` to `&lt;`)
2. Added Mathjax support to Javadocs

Javadocs can be generated with `./gradlew javadoc` on Linux/Mac or `gradlew.bat javadoc` on Windows.

### CODE REVIEW

**Comment by wilsonwatson** [comment]
Added as draft to discuss: should we make Javadocs generate a `github.io` page associated with this repo?

File: src/main/java/frc/lib/util/swerve/SwerveModule.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ private void resetToAbsolute() {
     /**
      *
      *
-     * @param rotationSpeed Drive motor speed (-1 <= value <= 1)
+     * @param rotationSpeed Drive motor speed (-1 &lt;= value &lt;= 1)
```
Comment (legoguy1000) [inline src/main/java/frc/lib/util/swerve/SwerveModule.java:106]: eww 🤣

**Comment by legoguy1000** [comment]
> Added as draft to discuss: should we make Javadocs generate a `github.io` page associated with this repo?
 
Lets do it!!!

**Comment by legoguy1000** [comment]
https://stackoverflow.com/questions/70991706/publish-automatically-javadoc-with-github-actions This uses the Branch method but i think it would be just as easy to use the GitHub Actions only mthod

File: src/main/java/frc/lib/util/swerve/SwerveModule.java (hunk 1 above)
Comment (wilsonwatson) [reply src/main/java/frc/lib/util/swerve/SwerveModule.java:106]: Note that in vscode, it automatically turns these into `<` for hovering. Only place this is ugly in the actual source code.

**Review by legoguy1000 — APPROVED** [review]
GTG

**Review by aidan-underwood — APPROVED** [review]
good

**Pull request merged**